docs/standard/tour.md

```diff
@@ -14,137 +14,132 @@ ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
 
 # Tour of .NET
 
-.NET is a general purpose development platform.  It has several key features, such as multiple programming languages, asynchronous and concurrent programming models, and native interoperability which enable a wide range of scenarios across multiple platforms.
+.NET is a general purpose development platform. It has several key features, such as multiple programming languages, asynchronous and concurrent programming models, and native interoperability which enable a wide range of scenarios across multiple platforms.
```

Comment:
Since .NET is language independent, better to say "support for multiple programming languages"